keyboards/kprepublic/bm80v2_iso/rules.mk

```diff
@@ -0,0 +1,22 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes        # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no          # Mouse keys
+EXTRAKEY_ENABLE = yes          # Audio control and System control
+CONSOLE_ENABLE = no            # Console for debug
+COMMAND_ENABLE = no            # Commands for debug and configuration
+NKRO_ENABLE = yes              # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no          # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no           # Enable keyboard RGB underglow
+AUDIO_ENABLE = no              # Audio output
+RGB_MATRIX_ENABLE = yes        # Use RGB matrix
+RGB_MATRIX_DRIVER = IS31FL3741
+LTO_ENABLE = yes
+LAYOUTS = tkl_iso
```

Comment:
Use of the Community Layouts feature for this board requires the layout macro and keymaps to be implemented such that the keycode for Enter should be placed toward the end of the Home row (after `KC_NUHS`), rather than on the row above (after `KC_RBRC`).

Was the submitted keymap tested on actual hardware?

Reply:
Yes. The keymap matches the hardware.